.vib/fluentd/goss/goss.yaml

```diff
@@ -0,0 +1,29 @@
+
+file:
+  {{ printf "/opt/bitnami/fluentd/conf/%s" .Env.FLUENTD_CONF }}:
```

Comment:
I think we could replace this test with a dry-run check of the config file using the `fluentd` binary as we do in BTS. The command is as follows:

```
$ fluentd -c /opt/bitnami/fluentd/conf/fluentd.conf --dry-run
2022-07-08 14:38:33 +0000 [info]: parsing config file is succeeded path="/opt/bitnami/fluentd/conf/fluentd.conf"
2022-07-08 14:38:34 +0000 [info]: gem 'fluentd' version '1.15.0'
...
```

Reply:
Applied this improvement!